### PR TITLE
Translation-ja: Add trusted-committer/ja/03-keeping-the-community-healthy

### PR DESCRIPTION
**Pull request by ystk**
This is the third PR for #367 .

### CODE REVIEW

File: trusted-committer/ja/03-keeping-the-community-healthy-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ここでも、Trusted Committer の役割は本当に重要です。
+Trusted Committers は多くの場合、ジュニア開発者のメンターとなり、プルリクエスト中に時間を割いて、改善すべき場所を指摘するだけでなく、なぜ改善が必要なのか、どのようにそれを行うのかの詳細も説明します。
+彼らは、変更の背景にある理論や経験を提供し、それを実装する最善の方法を提案します。
+そうすることで Trusted Committer は、コミュニティの学習スピードを、従来のソフトウェア開発プロジェクトをはるかに上回るものに高めることができます。
+
+私たちは、Trusted Committer は、よほど正当な理由がない限り、周知されたリリース日に到達するよりも、プルリクエスト中のオンボーディングと指導を優先すべきだと考えています。
+プルリクエスト中に優れた指導を行うことは、 https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] の信頼と関与のレベルを高め、より多くのコントリビューションにつながります。
+これについては、 https://innersourcecommons.org/resources/learningpath/trusted-committer/04/["コミュニティのレベル向上"] で詳しく説明します。
+
+最後に、InnerSource のコミュニティに留まる人たちもいますが、それは彼らがオーバーヘッドや無駄と考えられる活動の代わりにソフトウェアの開発に集中するようになったからで、これはプロセスに重点を置いている大企業では特によく見られるものです。
```
Comment (johtani) [inline trusted-committer/ja/03-keeping-the-community-healthy-ja.asciidoc:35]:
```suggestion
最後に、特にプロセスに重点を置いている大企業で、彼らがオーバーヘッドや無駄と考える活動の代わりに、ソフトウェアの開発に集中できるのでInnerSource のコミュニティに留まる人たちもいます。
```
ではどうでしょう？

File: trusted-committer/ja/03-keeping-the-community-healthy-ja.asciidoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+このコンテキストでの Trusted Committer の仕事は、有用なコントリビューションガイドラインを伝えて制定することにより、 https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] が実際にプロジェクトに集中できるようにすることです。
+
+これらのガイドラインの重要な側面の1つは、私たちがプルリクエストで _シグナリング_ と呼ぶものを説明することです。コメントはどのように見えるべきか?
+「いいね」や「+1」のコメントの意味は何ですか？
+/CC プレフィックスを付けて誰かを@メンションすることと、/FYI プレフィックスを使う場合はどのように違うのですか?
+一般的に、Trusted Committer はコントリビューションプロセスがより多くの問題を作らないことを確認する必要がありますが、それよりも問題の特定と解決でコミュニティをサポートします。
```
Comment (johtani) [inline trusted-committer/ja/03-keeping-the-community-healthy-ja.asciidoc:41]:
```suggestion
一般的に、Trusted Committer は、コントリビューションプロセスがより多くの問題を作らず、それよりも問題の特定と解決でコミュニティをサポートすることを確認する必要があります。
```
はどうでしょう？

**Review by johtani — CHANGES_REQUESTED** [review]
Hi @ystk , great work!
I left 3 suggestions, please check these.

**Comment by rrrutledge** [comment]
Awesome, @johtani❗  And since you are doing commit suggestions your name will show up on the website as a contributor!

**Comment by ystk** [comment]
Thanks for suggestions, @johtani ! I merged your suggested changes.

**Review by johtani — APPROVED** [review]
LGTM!

**Comment by rrrutledge** [comment]
Great work, both.  We'll get it on the website.

**Comment by rrrutledge** [comment]
Opened pull request for the website: https://github.com/InnerSourceCommons/innersourcecommons.org/pull/258.  Both of your names listed as contributing!

**Pull request merged**